pipenv/utils/resolver.py

```diff
@@ -25,6 +25,7 @@
 from pipenv.patched.pip._internal.req.req_install import InstallRequirement
 from pipenv.patched.pip._internal.utils.temp_dir import global_tempdir_manager
 from pipenv.patched.pip._vendor import pkg_resources, rich
+from pipenv.patched.pip._vendor.packaging.utils import canonicalize_name
 from pipenv.project import Project
 from pipenv.utils.fileutils import create_tracked_tempdir
 from pipenv.utils.requirements import normalize_name
```

Comment:
From my point of vie we should use canonicalize_name funiction from pip instead of normalize_name function everywhere.
WDYT?